examples/CMakeLists.txt

```diff
@@ -2,15 +2,16 @@
 ## I. Set project details
 #############################
 cmake_minimum_required(VERSION 3.14)
-set(CMAKE_POLICY_DEFAULT_CMP0127 NEW)
+#set(CMAKE_POLICY_DEFAULT_CMP0127 NEW)
```

Comment:
Should we remove this comment altogether?

```suggestion
```

Reply:
Hi Lee, as the CMP0127 warning message is due to `pybind11`, I think the correct place is the main `CMakeLists.txt` (if it should be present). Thus I move this line into the main `CMakeLists.txt` in the 296d79b74e82c2cc216f7c29ce1c060b28b8cea. Still, I hope to remove this line after moving to pybind11 v.2.8.1, which seems to resolve this issue (https://github.com/pybind/pybind11/releases/tag/v2.8.1).

Reply:
Sounds good!